### PR TITLE
Install styleguidist and write first component

### PR DESCRIPTION
**Pull request by macfrei**
Install and configure styleguidist.

### CODE REVIEW

File: styleguide.config.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  components: 'components/**/*.js',
+  webpackConfig: {
+    module: {
+      rules: [
+        // Babel loader will use your project’s babel.config.js
```
Comment (miriamhaenle) [inline styleguide.config.js:6]:
```suggestion
```

File: styleguide.config.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        {
+          test: /\.jsx?$/,
+          exclude: /node_modules/,
+          loader: 'babel-loader',
+        },
+        // Other loaders that are needed for your components
```
Comment (miriamhaenle) [inline styleguide.config.js:12]:
```suggestion
```

File: styleguide.config.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        {
+          test: /\.css$/,
+          use: ['style-loader', 'css-loader'],
+        },
```
Comment (miriamhaenle) [inline styleguide.config.js:16]: We should figure out if we need this css part here as we are planning on using styled components. But I think we can keep it until then and figure it out on the go .

**Review by miriamhaenle — CHANGES_REQUESTED** [review]

File: styleguide.config.js (hunk 3 above)
Comment (macfrei) [reply styleguide.config.js:16]: Good point!

File: styleguide.config.js (hunk 2 above)
Comment (macfrei) [reply styleguide.config.js:12]: True! 🤦‍♀️

**Pull request merged**